k8s-artifacts/k8s_deployments.yaml

```diff
@@ -0,0 +1,103 @@
+# Deployments for Adapter, router and enforcer
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: adapter
```

Comment:
Lets called these deployments as mg-adapter, mg-enforcer, mg-router. I think we can keep service as it is as we need to match these with docker compose artifacts.

Reply:
Fixed in 736f7ca09587d301737d3a6bff36ff140493ae4b